src/abstractions/RequestInformation.cs

```diff
@@ -492,5 +492,26 @@ public void SetContentFromScalar<T>(IRequestAdapter requestAdapter, string conte
             Headers.TryAdd(ContentTypeHeader, contentType);
             Content = writer.GetSerializedContent();
         }
+
+        /// <summary>
+        /// Sets the request body from an enum value with the specified content type.
+        /// </summary>
+        /// <param name="requestAdapter">The core service to get the serialization writer from.</param>
+        /// <param name="item">The enum value to serialize.</param>
+        /// <param name="contentType">The content type to set.</param>
+        /// <typeparam name="T">The enum type to serialize.</typeparam>
+#if NET5_0_OR_GREATER
+        public void SetContentFromEnum<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicFields)] T>(IRequestAdapter requestAdapter, string contentType, T? item) where T : struct, Enum
```

Comment:
quick question regarding the nullable item parameter: what was your reasoning for making it nullable?